cleancat/base.py

```diff
@@ -532,6 +618,8 @@ def full_clean(self):
 
                     self.data[field_name] = value
 
+            except ReferenceNotFoundError:
```

Comment:
I'd probably catch this in the `EmbeddedReference` implementation and throw a `ValidationError` from there, rather than handle this in the schema. We shouldn't have field-specific error handling in here.

Reply:
Done in 26b476a